client/src/partials/debtors/groups.list.html

```diff
@@ -0,0 +1,70 @@
+<div class="container text-center">
+<div class="row">
+<div class="col-md-8 col-md-offset-2">
+
+    <div class="panel panel-default" style="margin-bottom : 0px;">
```

Comment:
That's a good idea! 

This has been discussed https://github.com/IMA-WorldHealth/bhima-2.X/issues/270#issuecomment-209486393. To be built on in the future!